<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lasq_kssrr</class>
 <widget class="QDialog" name="lasq_kssrr">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lasq_kssrr</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="lasq_kssrr_xxprg_l">
     <property name="text">
      <string>=lasq_kssrr=</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="lasq_kssrr_exit_c">
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lasq_kssrr_xxlas_l">
     <property name="text">
      <string>=Луд А.С.=</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="lasq_kssrr_grid_c">
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lasq_kssrr_back_c">
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lasq_kssrr_next_c">
       <property name="text">
        <string>Вперед</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="lasq_kssrr_b1">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="7" column="0">
       <widget class="QLabel" name="lasq_kssrr_rubdb_l">
        <property name="text">
         <string>lasq_сумма Дб</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lasq_kssrr_rubdb_e"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lasq_kssrr_to_e"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lasq_kssrr_to_l">
        <property name="text">
         <string>lasq_операция</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lasq_kssrr_rub_l">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_data_e"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lasq_kssrr_dokk_l">
        <property name="text">
         <string>lasq_документ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_sae_e"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="lasq_kssrr_ks_l">
          <property name="text">
           <string>lasq_Корр счет</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_kr_e"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_ksn_e"/>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_s_e"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_sn_e"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lasq_kssrr_sae_l">
        <property name="text">
         <string>lasq_сае</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lasq_kssrr_data_l">
        <property name="text">
         <string>lasq_дата операции</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lasq_kssrr_s_l">
        <property name="text">
         <string>lasq_счет</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_dokk_e"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_dokn_e"/>
        </item>
        <item>
         <widget class="QLabel" name="lasq_kssrr_dokd_l">
          <property name="text">
           <string>от</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_dokd_e"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="7" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="lasq_kssrr_rubkr_l">
          <property name="text">
           <string>lasq_сумма Кр</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lasq_kssrr_rubkr_e"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lasq_kssrr_xxtitle_l">
     <property name="text">
      <string>lasq_книга счетов(КС)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lasq_kssrr_data_e</tabstop>
  <tabstop>lasq_kssrr_dokk_e</tabstop>
  <tabstop>lasq_kssrr_dokn_e</tabstop>
  <tabstop>lasq_kssrr_dokd_e</tabstop>
  <tabstop>lasq_kssrr_to_e</tabstop>
  <tabstop>lasq_kssrr_s_e</tabstop>
  <tabstop>lasq_kssrr_sn_e</tabstop>
  <tabstop>lasq_kssrr_kr_e</tabstop>
  <tabstop>lasq_kssrr_ksn_e</tabstop>
  <tabstop>lasq_kssrr_sae_e</tabstop>
  <tabstop>lasq_kssrr_rubdb_e</tabstop>
  <tabstop>lasq_kssrr_rubkr_e</tabstop>
  <tabstop>lasq_kssrr_grid_c</tabstop>
  <tabstop>lasq_kssrr_back_c</tabstop>
  <tabstop>lasq_kssrr_next_c</tabstop>
  <tabstop>lasq_kssrr_exit_c</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
